EKF/ekf.cpp

```diff
@@ -47,20 +47,14 @@
 bool Ekf::init(uint64_t timestamp)
 {
 	bool ret = initialise_interface(timestamp);
-	_state.vel.setZero();
-	_state.pos.setZero();
-	_state.gyro_bias.setZero();
-	_state.accel_bias.setZero();
-	_state.mag_I.setZero();
-	_state.mag_B.setZero();
-	_state.wind_vel.setZero();
-	_state.quat_nominal.setZero();
-	_state.quat_nominal(0) = 1.0f;
+	reset(timestamp);
```

Comment:
A comment here could be helpful

Reply:
It's basically THE ekf reset function, should I call it `resetAll` ?